checkClock/app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,270 @@
+@import 'variables.scss';
+@import 'clearStyles.scss';
+main{
+  align-items: start;
+  display: flex;
+  font-family: $poppins;
+  height: 100vh;
+  justify-content: center;
```

Comment:
After each selector and its properties add a blank line, this for best practices and for those who are code reviewing your code can read it better.

Reply:
This recommendation has already been added, thank you very much.